beacon-chain/p2p/gossip_scoring_params.go

```diff
@@ -215,6 +225,48 @@ func defaultAggregateTopicParams(activeValidators uint64) (*pubsub.TopicScorePar
 	}, nil
 }
 
+func defaultSyncContributionTopicParams() (*pubsub.TopicScoreParams, error) {
```

Comment:
I definitely feel like this is magic when I read it, and could benefit from being pointed to a source on how these were decided. Even explaining at the top of the file that these come from the altair spec and linking to the doc would be helpful

Reply:
For an overview of how the parameters were decided you can take a look here:
https://github.com/prysmaticlabs/prysm/pull/9390/files#diff-30b192f6a6c2c3197c073f0be4f714c3c14372b20c7e4d12f0b8bb4d1a236c5aR154

Unfortunately a lot of these parameters were based on empirical data and gossip sub spec parameters.  This follows the pattern from our attestation aggregate proof topic parameters here, a lot of the reasoning for the parameters are based on spec heuristics. Ex: Number of expected messages per slot, etc. It is difficult to put everything up into this file.